protocols/smtp/src/main/java/org/apache/james/protocols/smtp/SMTPConfiguration.java

```diff
@@ -36,6 +37,21 @@
  *
  */
 public interface SMTPConfiguration extends ProtocolConfiguration {
+    enum SenderVerificationMode {
+        STRICT,
+        RELAXED,
+        DISABLED;
+
+        // TODO unit tests
+        public static SenderVerificationMode parse(String value) {
+            return switch (value.toLowerCase(Locale.US).trim()) {
+                case "true", "strict" -> STRICT;
```

Comment:
maybe you will like util `org.apache.commons.lang3.BooleanUtils#toBoolean(java.lang.String)`

Reply:
Thanks for sharing.

It do not handle strict / disabled.